index.js

```diff
@@ -22,10 +23,16 @@ module.exports = function (opts) {
 			from: file.path,
 			to: file.path
 		}).then(function (res) {
+			var map;
 			file.contents = new Buffer(res.css);
 
 			if (res.map && file.sourceMap) {
-				applySourceMap(file, res.map.toString());
+				map = res.map.toJSON();
+				map.file = file.relative;
+				map.sources = [].map.call(map.sources, function (source) {
+					return path.join(path.dirname(file.relative), source);
+				});
```

Comment:
Sorry for being a bit unclear. Without these changes it's simply not working right now.

Right now it only keeps the autoprefixer sourcemaps, and throws away the LESS ones, it outputs this:

```
version: 3,
sources: [
   "top-navigation.css"
],
names: [ ],
mappings: "..",
file: "/Styles/Domain/Navigation/top-navigation.css",
sourcesContent: [
   "..."
],
sourceRoot: "/source/"
}
```

Giving this in DevTools, that's not useful at all since it's not the real source.
![image](https://cloud.githubusercontent.com/assets/453018/13600124/187c07ca-e527-11e5-993d-9f3a94c4f8bf.png)

After this patch it gives this:

```
version: 3,
sources: [
   "/Styles/Domain/Navigation/top-navigation.css",
   "/Styles/Domain/Navigation/top-navigation.less",
   "/Styles/TextMixins.less",
   "/Styles/BoxMixins.less"
],
names: [ ],
mappings: "..",
file: "/Styles/Domain/Navigation/top-navigation.css",
sourcesContent: [
   "...",
   "...",
   "...",
   "..."
],
sourceRoot: "/source/"
}
```
that's translated to the real source in DevTools:

![image](https://cloud.githubusercontent.com/assets/453018/13600187/89448e8c-e527-11e5-92d1-22f6f171fdac.png)

Hope this makes a bit more sense.